### PR TITLE
base-nl.yaml

### PR DESCRIPTION
**Pull request by ItzSpiro**
I changed a couple translations so they fit better and I made every translation that wasn't done yet. I hope this helped a lot since I put a lot of time in it.

### CODE REVIEW

File: translations/base-nl.yaml [hunk 1]
```diff
@@ -197,21 +198,20 @@ dialogs:
         desc: Je kunt het je niet veroorloven om de selectie te plakken! Weet je zeker
             dat je het wil knippen?
     editSignal:
-        title: Set Signal
+        title: Stel het signaal in.
         descItems: "Kies een ingesteld item:"
         descShortKey: ... of voer de <strong>short key</strong> van een vorm (Die je
```
Comment (LeopoldTal) [inline translations/base-nl.yaml:203]: You left "short key" untranslated here but translated it as "sleutel" [in the shape viewer](https://github.com/MisterPengu/shapez.io/blob/f8df08bfaf3e07a5773d45998f3c8cca32a304de/translations/base-nl.yaml#L347), should pick one and be consistent

File: translations/base-nl.yaml [hunk 2]
```diff
@@ -307,30 +307,30 @@ ingame:
...
-                don't have to connect all inputs! Try wiring only two."
+            2_1_place_cutter: "Plaats nu een <strong>Knipper</strong> om de cirkels in twee te knippen
+                halves!<br><br> PS: De knipper knipt altijd van <strong>boven naar
+                onder</strong> ongeacht zijn oriëntatie."
+            2_2_place_trash: De knipper kan vormen <strong>verstoppen en bijhouden</strong>!<br><br> Gebruik een
+                <strong>vuilbak</strong> om van het (!) niet
+                nodige afval vanaf te geraken.
+            2_3_more_cutters: "Goed gedaan! Plaats nu <strong>2 extra knippers</strong> om dit traag
+                process te versnellen! <br><br> PS: Gebruik de <strong>0-9
+                hotkeys</strong> om gebouwen sneller te selecteren."
```
Comment (LeopoldTal) [inline translations/base-nl.yaml:318]: Inconsistent: sometimes "sneltoets", sometimes "hotkey", pick one

**Review by LeopoldTal — COMMENTED** [review]
You should use no-break spaces where appropriate:

line break inside the phrase shapes per second

File: translations/base-nl.yaml [hunk 3]
```diff
@@ -723,24 +723,24 @@ storyRewards:
...
+        desc: Ik heb juist een hele hoop nieuwe gebouwen toegevoegd die je toetstaan om
+            <strong>het process van vormen te stimuleren</strong>!<br><br> Je kan
+            nu de knipper, draaier, stapelaar en meer op de dradenlaag stimuleren!
+            Met dit heb je nu 3 opties om verder te gaan met het spel:<br><br> -
+            Bouw een <strong>automatische fabriek</strong> om eender welke mogelijke
+            vorm te maken gebraagd door de HUB (Ik raad aan dit te proberen!).<br><br> - Bouw
+            iets cool met draden.<br><br> - Ga verder met normaal
+            spelen.<br><br> Wat je ook kiest, onthoud dat je plezier hoort te hebben!
     reward_wires_painter_and_levers:
         title: Wires & Quad Painter
```
Comment (LeopoldTal) [inline translations/base-nl.yaml:736]: Missing translation

**Comment by ItzSpiro** [comment]
> You should use no-break spaces where appropriate:
> 
> line break inside the phrase shapes per second

Hey, thanks for all those things i still need to change. I changed all of the things you told me so far so they are consistand and fit in the sentences. But i have no clue in what line that picture is.

**Comment by ItzSpiro** [comment]
> You should use no-break spaces where appropriate:
> 
> line break inside the phrase shapes per second

Hey, thanks for letting me know what could be better. I reviewed all of your things and changed where needed. Should be better now. But i am still searching for the picture above, that'll take some time before i find it. But i will change it asap.

**Comment by tobspr** [comment]
Thank you!

**Comment by tobspr** [comment]
If anything needs to fixed, just make a new PR! :)

**Pull request merged**